.build/external-documentation/cli-command-reference.quickstart.yml

```diff
@@ -4,6 +4,14 @@ metadata:
   name: docs-cli-command-reference
   annotations:
     order: 4
+  labels:
+    service-category/application-services: "true"
+    product/rhosak: "true"
+    product/rhosr: "true"    
+    product/rhoad: "true"
+    product/rhoam: "true"
```

Comment:
I think we might need to remove the rhoam entry here too, I think the CLI ref does not apply to that service -- @jbyrne-redhat can confirm.

Reply:
Yes, @bredamc  - similar to my comment in https://github.com/redhat-developer/app-services-guides/pull/617#discussion_r1095633832.